implementation/data.json

```diff
@@ -49,602 +32,448 @@
+            "liquidityIndex": "0.38008699179130556",
+            "collateralIndex": "0.9927140661481465"
         }
     },
     {
         "asset": "dai",
         "protocol": "aave",
         "metrics": {
-            "score": "6.6",
-            "liquidityIndex": "0.23771333804334718",
-            "collateralIndex": "0.15023685445643176"
+            "score": "6.9",
+            "liquidityIndex": "0.3726542025758556",
+            "collateralIndex": "0.31406386068626013"
         }
     },
     {
         "asset": "bat",
         "protocol": "aave",
         "metrics": {
-            "score": "7.4",
-            "liquidityIndex": "0.22945415661398832",
-            "collateralIndex": "0.9842308738006139"
+            "score": "7.5",
+            "liquidityIndex": "0.2613347656968942",
+            "collateralIndex": "0.9863965967487084"
         }
     },
     {
         "asset": "lend",
         "protocol": "aave",
         "metrics": {
-            "score": "7.6",
-            "liquidityIndex": "0.4734714885050531",
-            "collateralIndex": "0.903625864736977"
+            "score": "7.7",
+            "liquidityIndex": "0.5390248728301618",
+            "collateralIndex": "0.9701305245817321"
         }
     },
     {
         "asset": "mkr",
         "protocol": "aave",
         "metrics": {
-            "score": "7.5",
-            "liquidityIndex": "0.318395869128073",
-            "collateralIndex": "0.988570177308151"
+            "score": "7.6",
+            "liquidityIndex": "0.34688313143566507",
+            "collateralIndex": "0.9901367581952748"
         }
     },
     {
         "asset": "tusd",
         "protocol": "aave",
         "metrics": {
-            "score": "7.0",
-            "liquidityIndex": "0.2944494099397984",
-            "collateralIndex": "0.4650864731202252"
+            "score": "6.8",
+            "liquidityIndex": "0.2890720636500724",
+            "collateralIndex": "0.32562061607566417"
         }
     },
     {
         "asset": "usdt",
         "protocol": "aave",
         "metrics": {
-            "score": "7.5",
-            "liquidityIndex": "0.550264921019968",
-            "collateralIndex": "0.7158848688437611"
+            "score": "6.9",
+            "liquidityIndex": "0.47108282708875243",
+            "collateralIndex": "0.1954530861016215"
         }
     },
     {
         "asset": "snx",
         "protocol": "aave",
         "metrics": {
-            "score": "6.9",
-            "liquidityIndex": "0.16908479212663619",
-            "collateralIndex": "0.5440453544704522"
+            "score": "7.3",
+            "liquidityIndex": "0.3433427799790493",
+            "collateralIndex": "0.7889756186347942"
         }
     },
     {
         "asset": "susd",
         "protocol": "aave",
         "metrics": {
-            "score": "7.2",
-            "liquidityIndex": "0.2451472942213502",
-            "collateralIndex": "0.7394504501095933"
-=======
```

Comment:
Uhh embarassing. Must have been a bad merge

Reply:
Hehe I wasn’t going to say anything ;]